lms/envs/acceptance.py

```diff
@@ -136,6 +136,9 @@ def seed():
 
 FEATURES['ENABLE_FEEDBACK_SUBMISSION'] = False
 
+# Show new tabbed view for student dashboard
+FEATURES['ENABLE_TABBED_DASHBOARD'] = True
```

Comment:
Features are out. Configuration models are in. @cpennington can you link to docs, please?